### PR TITLE
Create messages_cs.po

### PR DESCRIPTION
**Pull request by Clonewayx**
init czech i18n

### CODE REVIEW

**Comment by mekarpeles** [comment]
Thank you for submitting this i18n set! It will be under review this week.

**Comment by roptat** [comment]
Hi, I'm considering writing a french translation, but first, could you generate a newer pot file? The current one is from 2010. Surely there was some changes since 8 years ago ;)

File: openlibrary/i18n/messages_cs.po [hunk 1]
```diff
@@ -0,0 +1,2179 @@
...
+msgstr "Smazat"
+
+#: macros/EditButtons.html:23
+msgid ""
+"Except where otherwise noted, content on this site is licensed under a <a"
+" href=\"http://creativecommons.org/choose/zero\">Creative Commons CC0 "
+"License"
+msgstr ""
+"Kde není uvedeno jinak,je obsah těchto stránek licencován <a"
+" href=\"http://creativecommons.org/choose/zero\">Creative Commons CC0"
```
Comment (LeadSongDog) [inline openlibrary/i18n/messages_cs.po]: Shouldn't this be https://creativecommons.org/choose/zero/?lang=cs instead?

**Comment by tfmorris** [comment]
@roptat Now that #801 has been merged, do you have what you need to work on a French translation?

File: openlibrary/i18n/messages_cs.po (hunk 1 above)
Comment (hornc) [reply openlibrary/i18n/messages_cs.po]: Good question to raise @LeadSongDog , I'm not sure it is necessary though if the user has browser language settings configured. I tested OL using Czech browser settings, and following the standard CC0 url was presented with some Czech (mainly navigation, the CC main content was still in English, but that's their issue).

I am loving the browser language setting option that @tfmorris brought to my attention. It means we do not have to code any custom settings pages or manage extra data. In addition to supporting my laziness, it does seem to be the way the web should work.

We need to decide (at some point) if OL needs prominent per session language links (which would advertise the feature), and account language preferences, but for now I am content with browser settings being the way to use and test i18n. I don't regularly use this sort of feature though, so data + opinions from users would be great to drive these decisions!

Comment (tfmorris) [reply openlibrary/i18n/messages_cs.po]: Browsers typically inherit from the operating system, extending the laziness. :-) My browser allows for an ordered list of fallback languages, which I think allows for a lot of flexibility/power without resorting to special case OL-specific code.

**Pull request merged**